deps/rabbitmq_stream/src/rabbit_stream_reader.erl

```diff
@@ -184,6 +99,9 @@
          tuned/3,
          open/3,
          close_sent/3]).
+%% for unit test
```

Comment:
Instead of using a comment, can we check for the TEST macro instead as done in https://github.com/rabbitmq/rabbitmq-server/blob/6a3ba6210aa831bc8a35deb8c432dbef1ca0decf/deps/rabbitmq_mqtt/src/rabbit_mqtt_processor.erl#L19-L23 ?
This will "codify" the comment and enforce that these functions are only called externally in tests.